docs/PRIVATE.md

```diff
@@ -54,12 +54,17 @@ return scopedKeys.deriveScopedKey({
 
 * [deriver-ScopedKeys](#module_deriver-ScopedKeys)
     * [~deriveScopedKey(options)](#module_deriver-ScopedKeys..deriveScopedKey) ⇒ <code>Promise</code>
+    * [~_deriveLegacySyncKey(options)](#module_deriver-ScopedKeys.._deriveLegacySyncKey) ⇒ <code>Promise</code> ℗
     * [~_deriveHKDF(salt, initialKeyingMaterial, info, keyLength)](#module_deriver-ScopedKeys.._deriveHKDF) ⇒ <code>Promise</code> ℗
 
 <a name="module_deriver-ScopedKeys..deriveScopedKey"></a>
 
 ### deriver-ScopedKeys~deriveScopedKey(options) ⇒ <code>Promise</code>
-Derive a scoped key
+Derive a scoped key.
+This method derives the key material for a particular scope from the user's master key material.
+For most scopes it will produce a JWK containing a 32-byte symmetric key.
+There is also special-case support for a legacy key-derivation algorithm used by Firefox Sync,
+which generates a 64-byte key when `options.identifier` is 'https://identity.mozilla.com/apps/oldsync'.
```

Comment:
@vladikoff I added some more words here, and I also marked the "deriveLegacySyncKey" helper as private because there's not really any reason to call it directly.  Thoughts?